packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1304,6 +1307,50 @@ export class ArcRotateCamera extends TargetCamera {
         super._updateRigCameras();
     }
 
+    /**
+     * @internal
+     */
+    public _calculateLowerRadiusFromModelBoundingSphere(
+        minimumWorld: Vector3,
+        maximumWorld: Vector3,
+        frustumSlope: Vector2 = this._getFrustumSlope(),
+        radiusScale: number = 1
+    ): number {
+        const size = maximumWorld.subtract(minimumWorld);
+        const boxVectorGlobalDiagonal = size.length();
+
+        // Formula for setting distance
+        // (Good explanation: http://stackoverflow.com/questions/2866350/move-camera-to-fit-3d-scene)
+        const radiusWithoutFraming = boxVectorGlobalDiagonal * 0.5;
+
+        // Horizon distance
+        const radius = radiusWithoutFraming * radiusScale;
+        const distanceForHorizontalFrustum = radius * Math.sqrt(1.0 + 1.0 / (frustumSlope.x * frustumSlope.x));
+        const distanceForVerticalFrustum = radius * Math.sqrt(1.0 + 1.0 / (frustumSlope.y * frustumSlope.y));
+        return Math.max(distanceForHorizontalFrustum, distanceForVerticalFrustum);
+    }
+
+    /**
+     * @internal
+     */
+    public _getFrustumSlope(): Vector2 {

```

Comment:
Maybe instead of a getter, it should be an updater of an existing (and cached) vector2? Since this is an internal function it is not a breaking change. It could be cached per frame and return the same object if called twice, but mainly will avoid GC.

Reply:
After reviewing the code, I found that, technically, I don't even need a `Vector2` for the slope.  I just need to do the math so I removed this function and move the slope math to the calculate function.  Should be able to avoid GC and not need to store a cached vector.